src/Bicep.Core.Samples/Files/InvalidModules_LF/main.diagnostics.bicep

```diff
@@ -292,22 +292,22 @@ module runtimeInvalidModule2 'empty.bicep' = {
 
 module runtimeInvalidModule3 'empty.bicep' = {
   name: runtimeValidRes1.sku.name
-//@[8:33) [BCP120 (Error)] The property "name" must be evaluable at the start of the deployment, and cannot depend on any values that have not yet been calculated. Accessible properties of runtimeValidRes1 are "apiVersion", "id", "name", "type". |runtimeValidRes1.sku.name|
+//@[8:28) [BCP120 (Error)] The property "name" must be evaluable at the start of the deployment, and cannot depend on any values that have not yet been calculated. Accessible properties of runtimeValidRes1 are "apiVersion", "id", "name", "type". |runtimeValidRes1.sku|
```

Comment:
The old visitor contains some ad-hoc logic to make sure the error on the full path. Personally I like the new error message better because essentially it is the top level property that triggers the error.